_data/government.yml

```diff
@@ -72,6 +72,12 @@ websites:
       - hardware
     doc: https://www.login.gov/help/creating-an-account/two-factor-authentication/
 
+  - name: Mi Argentina
+    url: https://id.argentina.gob.ar
```

Comment:
The url tag shouldn't link to a login page. Is there a main page or similar that you can use instead?

Reply:
The main page is https://mi.argentina.gob.ar/, but it redirects to https://id.argentina.gob.ar. As soon as you log in, you'll see your available options.

Reply:
Regarding the DNI on your phone, the idea of the government is that you don't need to carry your physical ID but instead use the "picture" of your ID that's inside "Mi Argentina" app. Banks and government offices just started to accepting this "digital" ID. And this is one of the reasons why I've been telling them to support a 2FA mechanism.

If anyone manages to get my username and password, that's it. They can get my ID, my drivers license, my medical information, and some other sensitive stuff.

Not sure if that clarifies your question. Let me know!

Reply:
Maybe link to https://www.argentina.gob.ar instead? There is a prominent "Mi Argentina" button at the top.

Reply:
Ha! That button is totally new. Makes sense, then. I'm pushing the updated URL.